### PR TITLE
[DOCU-2317] Konnect administer section clean-up and validation

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
* Style clean-up
* Adjust nav to follow nesting style (no links for top-level accordion items)
* Adjust instructions for new nav and UI updates
* Split up user list items into basic tasks

### Reason
https://konghq.atlassian.net/browse/DOCU-2317

### Testing
netlify TBA

### CODE REVIEW

File: app/konnect/org-management/teams-and-roles/index.md [hunk 1]
```diff
@@ -19,52 +19,50 @@ access.
...
-are additive. For example, if you add a user to both the Service Developer and
-Portal Viewer teams, the user can create and manage Services
+are additive. For example, if you add a user to both the `Service Developer` and
+`Portal Viewer` teams, the user can create and manage services
 through the Service Hub _and_ register applications through the Dev Portal.
 
 If two roles provide access to the same entity, the role with more access
-takes effect. For example, if you have the Service Admin and Service Deployer
-roles on the same Service, the Service Admin role takes precedence.
+takes effect. For example, if you have the `Service Admin` and `Service Deployer`
```
Comment (Guaris) [inline app/konnect/org-management/teams-and-roles/index.md:47]:
```suggestion
takes effect. For example, if you have the Service Admin and Service Deployer
```
I'm removing these because I think it is confusing to use code tags to mark something that doesn't have to do with code.

File: app/konnect/org-management/teams-and-roles/index.md [hunk 2]
```diff
@@ -19,52 +19,50 @@ access.
...
+<!-- (SHARING IS NOT YET AVAILABLE)
+### Entity and role sharing
 
-An Organization Admin can share any role or entity with any user in the
+An `Organization Admin` can share any role or entity with any user in the
 organization.
 
-Any user with the Service Admin or Runtime Group Admin role can
-also share Services or runtime groups that they have access to, with
+Any user with the `Service Admin` or `Runtime Group Admin` role can
```
Comment (Guaris) [inline app/konnect/org-management/teams-and-roles/index.md:56]:
```suggestion
Any user with the Service Admin or Runtime Group Admin role can
```

File: app/konnect/org-management/teams-and-roles/index.md [hunk 3]
```diff
@@ -19,52 +19,50 @@ access.
...
+also share services that they have access to, with
 the same role or lesser.
 
-For example, say you have a Service Admin role:
-* You can share that Service with any other user through the Service Hub.
-* Since you have admin access, you can choose to share the Service any
+For example, say you have a `Service Admin` role:
+* You can share that service with any other user through the Service Hub.
+* Since you have admin access, you can choose to share the service with any
```
[the remark this answers is not in the document]
Comment (Guaris) [reply app/konnect/org-management/teams-and-roles/index.md:62]: maybe too wordy.

File: app/konnect/org-management/teams-and-roles/manage.md [hunk 4]
```diff
@@ -31,51 +33,47 @@ configuration page.
...
     or see the [roles reference](/konnect/org-management/teams-and-roles/roles-reference).
 
-6. Click **Save**.
+1. Click **Save**.
 
 
 ### Add or remove team members
 
 Edit team membership for any team, custom or predefined.
+Team members gain all roles attributed to the team.
```
Comment (Guaris) [inline app/konnect/org-management/teams-and-roles/manage.md:55]:
```suggestion
Team members inherit all roles attributed to the team.
```

File: app/konnect/org-management/teams-and-roles/manage.md [hunk 5]
```diff
@@ -31,51 +33,47 @@ configuration page.
...
+1. Open {% konnect_icon organizations %} [**Organization**](https://cloud.konghq.com/organization/) to go to the **Teams** page, then choose a team from the list.
 
-The user gains all roles attributed to the team.
+1. On the **Members** tab, click **Add Member(s)**.
+
+1. Select a user in the organization and click **Save**.
 
 ### Delete a team
 
 Deleting a team completely removes it and its configuration from
```
Comment (Guaris) [inline app/konnect/org-management/teams-and-roles/manage.md:65]:
```suggestion
Deleting a team removes it and its configuration from
```

File: app/konnect/org-management/users.md [hunk 6]
```diff
@@ -44,20 +47,36 @@ automatically-created users, adjust user permissions through Okta, or
...
+1. From the **User Actions** drop-down menu, select **Add or remove teams** to change the user's team membership.
+
+1. Check or uncheck any teams, then click **Ok** to save.
+
+
+## Manage individual user roles
+
+1. From {% konnect_icon organizations %} [**Organization**](https://cloud.konghq.com/organization/), open the **Users** page.
+
+1. Select a user row to drill down to their assigned teams and roles.
```
Comment (Guaris) [inline app/konnect/org-management/users.md:65]:
```suggestion
1. Select a user row to view their assigned teams and roles.
```

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/konnect/org-management/teams-and-roles/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/konnect/org-management/teams-and-roles/index.md:62]: i'll add for now, and can take a close look when we publish the share docs (not currently going live anyway).

File: app/konnect/org-management/teams-and-roles/manage.md [hunk 7]
```diff
@@ -31,51 +33,47 @@ configuration page.
...
 
-1. From the left navigation menu in {{site.konnect_short_name}}, open the {% konnect_icon organizations %}
- **Organization** link.
-2. Choose a team from the list.
+1. Open {% konnect_icon organizations %} [**Organization**](https://cloud.konghq.com/organization/) to go to the **Teams** page, then choose a team from the list.
 
     Teams with the `Predefined` label can't be edited or deleted.
 
-3. Choose an entity tab, then click **+ Add role(s)**.
+1. Open either the **Services** or **Runtime Groups** tab, then click **Add role(s)**.
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/konnect/org-management/teams-and-roles/manage.md:40]: Hmm. I don't think either of those quite works, since it places "services" as more important.

Maybe this is an instance when I should actually split the step, ie:

> 1. Choose one of the entity tabs: Services or Runtime Groups.
> 2. Click **Add role**.

Going with something like that for now.

**Review by Guaris — APPROVED** [review]
